weblate/auth/models.py

```diff
@@ -183,15 +183,15 @@ def save(self, *args, **kwargs):
     @cached_property
     def short_name(self):
         if "@" in self.name:
-            return pgettext("Per project access control group", self.name.split("@")[1])
+            return pgettext("Per-project access-control group", self.name.split("@")[1])
         return self.__str__()
 
 
 class UserManager(BaseUserManager):
     use_in_migrations = True
 
     def _create_user(self, username, email, password, **extra_fields):
-        """Create and save a User with the given username, e-mail and password."""
+        """Create and save a user account with the given username, e-mail address and password."""
```

Comment:
It's User object, so I'd stick with original text:

```suggestion
        """Create and save a User with the given username, e-mail address and password."""
```